README.md

```diff
@@ -248,7 +251,7 @@ def test_emitted_full():
         capture_deferred_events=True,
         capture_framework_events=True,
     )
-    ctx.run(ctx.on.start(), scenario.State(deferred=[scenario.Event("update-status").deferred(MyCharm._foo)]))
+    ctx.run(ctx.on.start(), scenario.State(deferred=[scenario._Event("update-status").deferred(MyCharm._foo)]))
```

Comment:
Do we need to have a private reference `_Event` in the README, or can we do this another way?

Reply:
Fair point. Improving deferred is one of the last two PRs but I should be able to clean this up in the meantime.